src/haddock/libs/libhpc.py

```diff
@@ -138,29 +137,24 @@ class HPCScheduler:
     """Schedules tasks to run in HPC."""
 
     def __init__(
-            self,
-            task_list: list[CNSJob],
-            target_queue: str = HPCWorker_QUEUE_DEFAULT,
-            queue_limit: int = HPCWorker_QUEUE_LIMIT_DEFAULT,
-            concat: int = HPCScheduler_CONCAT_DEFAULT,
-            ) -> None:
+        self,
+        task_list: list[CNSJob],
+        target_queue: str = HPCWorker_QUEUE_DEFAULT,
+        queue_limit: int = HPCWorker_QUEUE_LIMIT_DEFAULT,
+        concat: int = HPCScheduler_CONCAT_DEFAULT,
+    ) -> None:
         self.num_tasks = len(task_list)
         self.queue_limit = queue_limit
         self.concat = concat
 
         # split tasks according to concat level
         if concat > 1:
             log.info(
-                f"Concatenating, each .job will produce {concat} "
-                "(or less) models"
-                )
-        job_list = [
-            task_list[i:i + concat] for i in range(0, len(task_list), concat)
-            ]
+                f"Concatenating, each .job will produce {concat} " "(or less) models"
```

Comment:
I think previous indentation was better:
- not space between two "string"
- less than 80 char per line
```suggestion
                f"Concatenating, each .job will produce {concat} "
                "(or less) models"
```